folder-structure.md

```diff
@@ -0,0 +1,209 @@
+# Folder Structue
+
+```
+├───webpack-cli
+│   ├───.github                                  ## Contains files required by Github for proper workflow
```

Comment:
This comment on the left is not present inside your script

Reply:
Done